docs/java-configuration.md

```diff
@@ -4,6 +4,91 @@ title: Scope Java Agent configuration
 sidebar_label: Configuration
 ---
 
+The behaviour of the Scope Agent can be modified by using  environment variables or by using a configuration file `Scope.yml`in the source root of the project.
+
+> If these properties are manually configured, they will be `true` only on encountering the string `true` configured on the environment variable. Any other value will be considered as `false`.
+
+## Disabling Scope instrumentation
+
+You can disable Scope auto instrumentation. Scope auto instruments your code and tests by default, you can disable this behaviour.
+
+### Using Environment Variables
+
+```sh
+SCOPE_AUTO_INSTRUMENT=false
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  auto_instrument: false
+  
+```
+
+
+
+## Changing service name
+
+You can change the name of the service when sending data to Scope. Scope initially uses `default` for the service name.
+
+### Using Environment Variables
+
+```sh
+SCOPE_SERVICE="service name"
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  service: 'service name'
+```
+
+
+
+## Setting Scope as Global Tracer
+
+You can set Scope as the Global Tracer of Open Tracing. Scope uses Open Tracing internally to handle Span and traces, using its own tracer, You can set it as the global tracer so it will capture all the spans generated by any Open Tracing library in relation to your tests. 
+
+### Using Environment Variables
+
+```sh
+SCOPE_SET_GLOBAL_TRACER=true
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  set_global_tracer: true
+  
+```
+
+
+
+## Change testing mode
+
+These variable indicates to the agent if it is running tests (`true`) or it is being used for runtime instrumentation (`false`). By default it will run in testing mode
+
+### Using Environment Variables
+
+```sh
+SCOPE_TESTING_MODE=false
+```
+
+### Using YAML Configuration file
+
+```yaml
+scope:
+  testing_mode: false
```

Comment:
Right now in Java is:
```yaml
scope:
  mode:
    testing: true
```

Should Java agent change it?

Reply:
Changed in all docs